### DISCUSSION TITLE
Retro and Learnings from 07/26/2 Hack Night

### DISCUSSION THREAD

**Discussion by Joshuarainwater** [KCWeb3/Organization#2] [Q&A]
@malcolmry00 @penguinistakc and anyone else, as we look back on yesterday what were some consequences / outcomes / effects we noticed that we noticed or that caught our attention? What:

Seemed to work well for us?
Seemed to not work well for us?
What were the learnings / lessons you think?
What tangible things might we want to do differently next time?
@DCLAW Feel free to give your two cents after we've had a few answers!

https://www.loom.com/share/fb27e23c865f4881bb6f573d26fc0355

**Comment by Joshuarainwater** [comment]
**Seemed to work well for us? Why?**

- Providing our discord channel, We were able to add 5 or so discord contributors because we had that to capture
- Separating into a developer option and a buisness option, I received good feedback and most ended up attending developer side
- Having a better spread of food, I received positive feedback on this
- Lanyards, seemed to give people added energy distinguished interests and professional back grounds  as well as more professional feel
- Sign up sheets since they allow us to track how many folks are coming to these and other helpful growth metrics

**Seemed not to work well for us? Why?**

- Generally working on things after 6, I was not able to connect with people as much
- Finishing lanyards and sign in sheets day of, spent a lot of time during event on this and stressed during the day
- Looking HDMI cords and other technical issues last minute, we did not have these ready and couldn't move parts of event forward
- Technical pieces - trouble with video software and video set up,  inhibited smooth flow of the event
- Sign in sheet - wasn't readable
- Lanyard design 
  - Lanyards are vertical but design was horizontal
  - Design required white markers since background was dark but this was needed last minute and none were found
  - Paper size was not correct fit to the lanyard card holder
-  Communication / alignment with programming hack side was lacking, result was no actual programming was accomplished as I hoped for

**What were the learnings / lessons you think?**

Generally striving to make these events higher in quality is the right direction to go in right now. My event planning skills are currently in need of help or more  learning about industry standard practices for hackathons and events. People really love that these events exist. We are moving the right direction. We need better project management, and ways of connecting people to tasks that anyone can do. It may actually be a good idea for me to step back from operations on the ground level to do more coordinating between folks. 

**What tangible things might we want to do differently next time?**

1. Begin forming an operating systems / rhythm structure - write a workflow for these events with check lists etc. 
2. Once all known tasks are created as issues, specifically ask people if they can do them and assign them on github with resources needed etc 
3. Make a due date for the day before the event for all things that can be done before it 
4. Arrival 45-30 min before start to help Ryan with space and technical aspects - see what can be done by others in this time

**Reply by malcolmry00** [reply]
- We need One Meetup page for both tracks.
- It should list common events, then list the working groups.
-MasterMind should go forward when a group coalesces more. Business Hack nights are good or Industry focus each night. TBD.
-Time for Working Groups for Algorand-KC. Mentorships and others.
-KUBC is great. That is an asset for our organization. They give different insights. Perhaps we should pursue more student clubs to create regional collaboration and possible enthusiastic help.


Yes, the event skills need to elevate. 
- There should always be someone to greet guests. Very important for first time people. 
-New Cards need to be made with simple logo and input boxes. Semi-permanence. (Future sticker stuff perhaps)
-Yes, Food is better.

-i find myself in too much operations and unable to speak with members.
- - - creates image of unapproachable
- - - need to get in touch with members more


We need a marketer. Please take pictures 

Do we have a website with pictures and calendar? Is Meetup considered the same? Honestly do not know.

We need donations, time and currency.
